src/main/java/com/redhat/devtools/lsp4ij/settings/ui/LanguageServerPanel.java

```diff
@@ -250,15 +260,21 @@ private static JLabel createLabelForComponent(@NotNull @NlsContexts.Label String
     }
 
     private void createConfigurationField(FormBuilder builder) {
-        configurationWidget = new LanguageServerConfigurationWidget(project);
+        configurationWidget = new JsonTextField(project);

```

Comment:
I removed the explcit subclasses of `JsonTextField` as they weren't adding any real value.